asteroid/data/librimix_dataset.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import soundfile as sf
+import torch
+from torch.utils.data.dataset import Dataset
+
+
+class LibriMix(Dataset):
+    """" Dataset class for Librimix source separation tasks.
+
+    Args:
+        metadata_path (str): The path to the metatdata file in
+        dataset/metadata
+        desired_length (int) : The desired sources and mixtures length in s
+        sample_rate (int) : The sample rate of the sources and mixtures
+        n_src (int) : The number of sources in the mixture
+    """
+
+    def __init__(self, metadata_path, desired_length=None, sample_rate=16000,
```

Comment:
Let's keep the same interface a the other datasets and have a `segment` argument instead of `desired_length`.